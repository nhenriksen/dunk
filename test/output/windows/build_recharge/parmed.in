parm init.recharge.prmtop
loadcoordinates init.recharge.rst7
timerge :1 :2 :1 :2
parmout recharge.prmtop
writecoordinates recharge.rst7
go
                