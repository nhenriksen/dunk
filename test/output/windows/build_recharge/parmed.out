
                        Y O U      H A V E      B E E N
                               _
                            ==(W{==========-      /===-
                              ||  (.--.)         /===-_---~~~~~~~~~------____
                              | \_,|**|,__      |===-~___                _,-' `
                 -==\\        `\ ' `--'   ),    `//~\\   ~~~~`---.___.-~~
             ______-==|        /`\_. .__/\ \    | |  \\           _-~`
       __--~~~  ,-/-==\\      (   | .  |~~~~|   | |   `\        ,'
    _-~       /'    |  \\     )__/==0==-\<>/   / /      \      /
  .'        /       |   \\      /~\___/~~\/  /' /        \   /'
 /  ____  /         |    \`\.__/-~~   \  |_/'  /          \/'
/-'~    ~~~~~---__  |     ~-/~         ( )   /'        _--~`
                  \_|      /        _) | ;  ),   __--~~
                    '~~--_/      _-~/- |/ \   '-~ \             
                   {\__--_/}    / \\_>-|)<__\      \
                   /'   (_/  _-~  | |__>--<__|      | 
                  |   _/) )-~     | |__>--<__|      |   
                  / /~ ,_/       / /__>---<__/      |  
                 o-o _//        /-~_>---<__-~      / 
                 (^(~          /~_>---<__-      _-~              
                ,/|           /__>--<__/     _-~                 
             ,//('(          |__>--<__|     /                  .----_ 
            ( ( '))          |__>--<__|    |                 /' _---_~\
         `-)) )) (           |__>--<__|    |               /'  /     ~\`\
        ,/,'//( (             \__>--<__\    \            /'  //        ||
      ,( ( ((, ))              ~-__>--<_~-_  ~--____---~' _/'/        /'
    `~/  )` ) ,/|                 ~-_~>--<_/-__       __-~ _/ 
  ._-~//( )/ )) `                    ~~-'_/_/ /~~~~~~~__--~ 
   ;'( ')/ ,)(                              ~~~~~~~~~~ 
  ' ') '( (/
    '   '  `             
                                  P A R M E D

ParmEd: a Parameter file Editor


['parmed.in']
Reading actions from parmed.in

Adding prmtop init.recharge.prmtop to parm list. init.recharge.prmtop is the active parm.
Adding coordinates to init.recharge.prmtop from init.recharge.rst7
Merging molecules [:1] [:2] with sc mask [:1] [:2]
Merging molecules :1 and :2 into the same molecule.
Use softcore mask:
timask1='@1,2,3,4,5,6,7,8,9,10,11,12,13',
timask2='@14,15,16,17,18,19,20,21,22,23,24,25,26',
scmask1='@1,2,3,4,5,6,7,8,9,10,11,12,13',
scmask2='@14,15,16,17,18,19,20,21,22,23,24,25,26',
Writing coordinates to recharge.rst7 as type RESTART
Outputting Amber topology file recharge.prmtop
Done!
